app/redux/Home/HomeActions.tsx

```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { AUTH_KEY, baseURL } from '../../services/ApiConfig';
+import { ApiErrorType, ApiResponseType } from '../../services/Types';
+import type { TrendingApiRequestParams } from './HomeTypes';
+
+export const getTrendingData = createAsyncThunk<
+  ApiResponseType,
+  TrendingApiRequestParams,
+  { rejectValue: ApiErrorType }
+>('trending', async (trendingData: TrendingApiRequestParams, thunkApi) => {
+  const { time, type } = trendingData;
+  try {
+    let response = await fetch(
+      baseURL + 'trending/' + type + '/' + time + '?api_key=' + AUTH_KEY,
```

Comment:
used string concatenation instead of plus. like `trending/${type}/${time}?api_key=${AUTH_KEY}`.